pkg/pool-weighted/test/IndexPool.test.ts

```diff
@@ -27,10 +27,10 @@ const getTimeForWeightChange = (weightDifference: number) => {
 };
 
 describe('IndexPool', function () {
-  let owner: SignerWithAddress, other: SignerWithAddress, vault: Vault;
+  let controller: SignerWithAddress, other: SignerWithAddress, vault: Vault;
 
   before('setup signers', async () => {
-    [, owner, other] = await ethers.getSigners();
+    [, controller, other] = await ethers.getSigners();
```

Comment:
Renaming this here will break a lot of tests in other PRs. Could you maybe just let it be called `owner` and assign it to another variable which is called `controller` so that we have two variables pointing at the same signer?

Reply:
Changed it back. We do have to merge this PR at some point tho because it does introduces some changes that are never present in the newer PR :D

Reply:
agreed, we should merge it asap